docs-src/src/pages/components/Slider.js

```diff
@@ -0,0 +1,38 @@
+import React, {Component} from 'react'
+import ExampleSection from '../../ExampleSection'
+import scope from '../../ExampleScope'
+import Headers from '../../Headers'
+
+const examples = {
+  Slider: require('raw!../../examples/Slider.js.example'),
+ }
```

Comment:
nit, there's a little extra whitespace here, the brace should line up w/ `const`